### PR TITLE
fix: curvature issue

### PR DESCRIPTION
**Pull request by xusiyuan841028**
The formula of bezier curve is wrong, please refer to [https://en.wikipedia.org/wiki/Radius_of_curvature#Definition](https://en.wikipedia.org/wiki/Radius_of_curvature#Definition)
ca3a739e9c10161f4b69598e5162ebb8

### CODE REVIEW

**Comment by xusiyuan841028** [comment]
@Pomax Can you help to review the fix and merge PR?

File: index.html [hunk 1]
```diff
@@ -479,9 +479,11 @@ <h2>This is an interactive API</h2>
                   let p = curve.get(t);
                   let n = curve.normal(t);
                   let kr = curve.curvature(t);
+                  console.log(Math.abs(kr.k))
```
Comment (Pomax) [inline index.html]: while I'm sure it was useful for debugging, please don't leave console logs in PR code =)

File: index.html [hunk 2]
```diff
@@ -461,8 +461,8 @@ <h2>This is an interactive API</h2>
                   let n = curve.normal(t);
                   let kr = curve.curvature(t);
                   drawLine(p, {
-                    x: p.x + n.x * kr.k,
-                    y: p.y + n.y * kr.k
+                    x: p.x + n.x * kr.r,
```
Comment (Pomax) [inline index.html]: absolutely not: this graphic is showing off the curvature, not the _radius_ of curvature. The tighter the curve, the larger the curvature value.

File: index.html [hunk 3]
```diff
@@ -493,7 +495,7 @@ <h2>This is an interactive API</h2>
           <p>
           <br><pre>         |     x'y" - y'x"    |
   κ(t) = | ------------------ |
-         |  (x'² + y'²)^(2/3) |</pre><br>
+         |  (x'² + y'²)^(3/2) |</pre><br>
```
Comment (Pomax) [inline index.html]: good catch

File: index.html [hunk 4]
```diff
@@ -479,9 +479,11 @@ <h2>This is an interactive API</h2>
                   let p = curve.get(t);
                   let n = curve.normal(t);
                   let kr = curve.curvature(t);
+                  console.log(Math.abs(kr.k))
+                  if (Math.abs(kr.k) <= 0.015) continue;
                   drawLine(p, {
-                    x: p.x + n.x * kr.k,
-                    y: p.y + n.y * kr.k
+                    x: p.x + n.x * kr.r,
```
Comment (Pomax) [inline index.html]: same as above: please undo this change.

File: index.html [hunk 5]
```diff
@@ -479,9 +479,11 @@ <h2>This is an interactive API</h2>
                   let p = curve.get(t);
                   let n = curve.normal(t);
                   let kr = curve.curvature(t);
+                  console.log(Math.abs(kr.k))
+                  if (Math.abs(kr.k) <= 0.015) continue;
```
Comment (Pomax) [inline index.html]: this optimization is not necessary for demonstrator code.

File: index.html (hunk 1 above)
Comment (xusiyuan841028) [reply index.html]: Has removed!

File: index.html (hunk 2 above)
Comment (xusiyuan841028) [reply index.html]: I replace `kr.k` with `kr.r`, because the results of `n.x * kr.k` are almost less than 1. So there are not any graphic on the most interpolation points .....

Use: `n.x * kr.k`
image

Use: `n.x * kr.r * 10000`
image

File: index.html (hunk 5 above)
Comment (xusiyuan841028) [reply index.html]: Remove it

**Comment by xusiyuan841028** [comment]
@Pomax Fix it!

**Comment by xusiyuan841028** [comment]
The rendering result is as following:
image

**Comment by Pomax** [comment]
perfect, thank you. In it goes.

**Comment by Pomax** [comment]
And v2.4 pushed
